packages/wonder-blocks-button/components/button.md

```diff
@@ -20,3 +20,127 @@ Disabled button example:
     disabled={true}
 >Label</Button>
 ```
+
+Button examples, `style` specified (Support width, position, margin, and flex styles):
+```js
+const {View} = require("wonder-blocks-core");
+<table>
+    <thead>
+        <tr>
+            <th style={{minWidth: '250px'}}>Styles</th>
+            <th style={{width: '100%'}}>Buttons</th>
+        </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td>width: '200px'</td>
+        <td>
+            <Button
+                onClick={(e) => console.log("Hello, world!")}
+                style={{width: '200px'}}
+            >Label</Button>
+        </td>
+    </tr>
+    <tr>
+        <td>width: '75%'</td>
+        <td>
+            <Button
+                onClick={(e) => console.log("Hello, world!")}
+                style={{width: '75%'}}
+            >Label</Button>
+        </td>
+    </tr>
+    <tr>
+        <td>display: 'block', margin: '0 auto'</td>
+        <td>
+            <Button
+                onClick={(e) => console.log("Hello, world!")}
+                style={{display: 'block', margin: '0 auto'}}
+            >Label</Button>
+        </td>
+    </tr>
+    <tr>
+        <td>flexGrow: 1</td>
+        <td>
+            <div style={{display: 'flex'}}>
+                <Button
+                    onClick={(e) => console.log("Hello, world!")}
+                    style={{flexGrow: 1}}
+                >Label</Button>
+            </div>
+        </td>
+    </tr>
+    <tr>
+        <td>flexShrink: 2, width: '300px'</td>
+        <td>
+            <div style={{display: 'flex'}}>
+                <Button
+                    onClick={(e) => console.log("Hello, world!")}
+                    style={{flexShrink: 2, width: '300px'}}
+                >Label</Button>
+                <div
+                    onClick={(e) => console.log("Hello, world!")}
+                    style={
```

Comment:
Instead of having a clickable "wide div", make it a button instead.  Since these two buttons are right beside each could you add some margin to one of them.  There are a number of layouts where we have buttons side by side with gaps and having such an example will give people confidence that they'll be able to do the same in their layouts.

Reply:
If this is just a wide div for layout purposes we can also just remove the `onClick` handler.